client/ayon_maya/api/plugin.py

```diff
@@ -992,6 +992,11 @@ def remove(self, container):
         """
         from maya import cmds
 
+        locknode_namespace = container["namespace"]
+        # check if the node has been locked and unlocked it
+        if cmds.lockNode(f"*{locknode_namespace}", query=True, lock=True):
+            cmds.lockNode(f"*{locknode_namespace}", lock=False)
```

Comment:
I'm so confused about this - **what** set are we referring to here? The animation instance? Why can't we be more accurate and **actually** find the instance correctly instead of wildcarding with a name (that is actually intended  to indicate a namespace?)